### PR TITLE
chore(deps): Upgrade all dev and production dependencies

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

File: src/Main.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
-// Not needed unless working with non "en" locales
-// import { addLocaleData } from 'react-intl';
-// import enLocaleData from 'react-intl/locale-data/en';
+// Box Elements uses react-intl for internationalization of our text strings and dates.
```
Comment (jstoffan) [inline src/Main.js]: Change `our` to `its`.

File: src/Main.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+// If you are not using a modern browser, you will have to polyfill parts of Intl.
+// See https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#runtime-requirements
```
Comment (jstoffan) [inline src/Main.js]: Make this a textual link:

>See the [formatjs runtime documentation](https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#runtime-requirements) for more details...

File: src/Main.js [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
 
-import React from 'react';
+// Polyfill Intl.PluralRules by uncommenting the following line
+// import '@formatjs/intl-pluralrules/dist-es6/polyfill';
+// Polyfill Intl.RelativeTimeFormat by uncommenting the following line
+// import '@formatjs/intl-relativetimeformat/dist-es6/polyfill';
+
+import * as React from 'react';
+import { IntlProvider } from 'react-intl';
+// The content explorer box element. Others can be imported similarly
```
Comment (jstoffan) [inline src/Main.js]: > The Content Explorer element

File: src/Main.js (hunk 2 above)
Comment (priyajeet) [reply src/Main.js]: Its JS

**Pull request merged**